src/webpack.config.prod.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const baseConfig = require('./webpack.config.js');
+
+
+module.exports = Object.assign({}, baseConfig, {
+  entry: [
+    path.join(__dirname, './index.jsx')
+  ],
+  output: {
+    filename: 'index.js',
+    library: 'react-cosmos-docs-proxy',
+    libraryTarget: 'commonjs2',
+    path: path.join(__dirname, '..', 'build') // where to place webpack files
+  },
+  module: {
+    rules: [{
```

Comment:
Could we copy these from the base config? Does anything needs to be changed for the prod config?